### DISCUSSION TITLE
While loop keeps printing 111111111.......n times

### DISCUSSION THREAD

**Discussion by lucifer1303** [dotnet/runtime#39083] [General] [answered]
while loop keeps printing 1..
if i change d += 1, it prints 1,2,3,4,5
but if it's d= +1, it keeps printing 1111.....
"
d = 1
while d<6:
    print(d)
    d= +1
"

**Comment by stephentoub** [comment, marked as answer]
`d= +1;` is the same as `d = 1;`.  It's an assignment, not a compound addition and assignment.

**Reply by lucifer1303** [reply]
Thanks a lot
